components/ContentBar/ContentBar.jsx

```diff
@@ -5,24 +5,10 @@ import {
   Container, Header, Content, Card, CardItem, Thumbnail, Text, Button, Icon,
   Left, Label, Body, Right, Title, Form, Input, Item, Spinner
 } from 'native-base';
-import { StyleSheet } from "react-native";
+import styles from "./ContentBarStyle";
 
 import CreateResourceModal from '../CreateResourceModal/CreateResourceModal';
 
-const { height, width } = Dimensions.get('window');
-
-const styles = StyleSheet.create({
-  bar: {
```

Comment:
Is this moved to somewhere else? It looks like the bar style is still being used on line 61

Reply:
@scholvat  `ContentBarStyle.js`, as imported above (and modified to carry over the style items described here). The pattern we've been using to to keep styles in adjacent files rather than creating the stylesheets in the component file itself.

Reply:
(duplicated because it didn't show up in this dialogue): ContentBarStyle.js, as imported above (and modified to carry over the style items described here). The pattern we've been using to to keep styles in adjacent files rather than creating the stylesheets in the component file itself.